mlos_bench/mlos_bench/service/__init__.py

```diff
@@ -8,14 +8,11 @@
 
 from mlos_bench.service.base_service import Service
 from mlos_bench.service.base_fileshare import FileShareService
-
 from mlos_bench.service.local.local_exec import LocalExecService
-from mlos_bench.service.config_persistence import ConfigPersistenceService
```

Comment:
Including `ConfigPersistenceService` here was initially causing circular import issues.
That said, I think I change all internal items that weren't using explicit file level imports so that they should all now be of the form:
```python
from mlos_bench.service.local.local_exec import LocalExecService
```
and not
```python
from mlos_bench.service import LocalExecService
```

Reply:
agree. the users won't use that service directly anyway